.idea/copyright/Mozilla.xml

```diff
@@ -1,6 +0,0 @@
-<component name="CopyrightManager">
-  <copyright>
-    <option name="myName" value="Mozilla" />
-    <option name="notice" value="This Source Code Form is subject to the terms of the Mozilla Public&#10;License, v. 2.0. If a copy of the MPL was not distributed with this&#10;file, You can obtain one at http://mozilla.org/MPL/2.0/." />
-  </copyright>
-</component>
```

Comment:
We rather like these `.idea` files. Could you unstage them from your PR?

Reply:
Removed